gapis/perfetto/android/trace.go

```diff
@@ -62,6 +62,10 @@ func hasRenderStageEnabled(perfettoConfig *perfetto_pb.TraceConfig) bool {
 
 // SetupProfileLayersSource configures the device to allow packages to be used as layer sources for profiling
 func SetupProfileLayersSource(ctx context.Context, d adb.Device, packageName string, abi *device.ABI) (app.Cleanup, error) {
+	cleanup, err := adb.SetupPrereleaseDriver(ctx, d, packageName)
```

Comment:
If you want the entire android.InstalledPackage here, you can get that for GAPIR from gapidapk.EnsureInstalled(...).apk

May be more replumbing than you want to do though..